crates/ironrdp-session/src/x224/mod.rs

```diff
@@ -90,43 +100,76 @@ impl Processor {
         process_svc_messages(messages.into(), channel_id, self.user_channel_id)
     }
 
-    /// Processes a received PDU. Returns a buffer with encoded data to send to the server, if any.
-    pub fn process(&mut self, frame: &[u8]) -> SessionResult<Vec<u8>> {
+    /// Processes a received PDU. Returns a vector of [`ProcessorOutput`] that should be processed
+    /// by the caller in orderly fashion.
```

Comment:
question: Did you mean "timely fashion"?

Reply:
Nope, I meant that the caller should process returned `ProcessorOutput`'s in order. Maybe I need to remove "in an orderly fashion." as it should be obvious from the returned type (Vec)?

Reply:
It seemed obvious to me, but it’s not bad to mention it. Maybe it could be said like this?

> a vector of [`ProcessorOutput`] that must be processed in the returned order.

Reply:
Yup, sounds good